docs-kits/kits/Manufacturing as a Service Kit/page_adoption-view.md

```diff
@@ -11,140 +11,94 @@ sidebar_position: 2
-value-added services such as automated search and matchmaking.
-Link to the semantic data
-model: <https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.manufacturing_capability/2.0.0>
+An aspect model representing manufacturing capabilities, based on the concepts for products, processes, resources and capabilities, as well as their relations to each other. Sharing information about the required and available manufacturing capabilities in an interoperable manner is fundamental for any MaaS solution providing basic and value-added services such as automated search and matchmaking.
+
+Link to the semantic data model: <https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.manufacturing_capability/2.0.0>
 
 ### Request for QuotationModel
 
-An aspect model defining detailed requirements, deadlines and evaluation criteria for obtaining quotations from
-potential manufacturers for specific products or services. Manufacturing-as-a-Service (MaaS) scenarios focus on
-connecting buyers having a request for specific manufacturing process steps or products to be manufactured with the
-appropriate manufacturing supplier, who has the corresponding capabilities and resources. Sharing information about the
-demand with all required configuration and contact data is necessary for potential suppliers to evaluate the request and
-formulate an offer. A common description of the request for quotation based on a standardized semantic definition is
-therefore key for facilitating such an information exchange between Catena-X participants. This ensures an open network
-for every Catena-X member to join and enables interoperability between the partners.
-Link to the semantic data
-model: <https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.request_for_quotation/2.0.0>
+An aspect model defining detailed requirements, deadlines and evaluation criteria for obtaining quotations from potential manufacturers for specific products or services. Manufacturing-as-a-Service (MaaS) scenarios focus on connecting buyers having a request for specific manufacturing process steps or products to be manufactured with the appropriate manufacturing supplier, who has the corresponding capabilities and resources. Sharing information about the demand with all required configuration and contact data is necessary for potential suppliers to evaluate the request and formulate an offer. A common description of the request for quotation based on a standardized semantic definition is therefore key for facilitating such an information exchange between Catena-X participants. This ensures an open network for every Catena-X member to join and enables interoperability between the partners.
+
+Link to the semantic data model: <https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.request_for_quotation/2.0.0>
```

Comment:
This should be aligned with CX-0129 Request for Quotation (v2.0.0), as it refers to  urn:samm:io.catenax.request_for_quotation:3.0.0 as the data model for R24.05.

Edit: The data model v3.0.0 seems already to be references in the other sections of this KIT, too.

Reply:
model references have been updated